_health-care/_js/_form.js

```diff
@@ -1241,6 +1285,9 @@ var HealthApp = (function() {
           e.preventDefault();
           // TODO(awong): Remove this debugging logic when things work.
           console.log(build1010ezXml(formRoot));
+          for (var i = 0; i < allInputs.length; i++) {
+            validate(allInputs[i]);
```

Comment:
Since we know the originating element for `change` though, we don't need to search for validation right?

If we move the handler down into the continue button, the only difference is how far up the DOM tree the event has to bubble, which I think is only going to be <10 elements in general. In fact, having it in the change handler lets us have immediate feedback w/o any searching of the DOM.